### PR TITLE
Added disabled state for individual menu items

### PR DESCRIPTION
**Pull request by duck-nukem**
#9 

I've created this PR to enable disabling individual menu-items.
I'd suggest adding tests and maybe creating a demo application inside the repo for others to be able to test their changes, and ensure they break nothing.

### CODE REVIEW

File: components/ngc-float-item-button.component.ts [hunk 1]
```diff
@@ -78,6 +84,11 @@ export class NgcFloatItemButtonComponent {
   @Input() icon;
   @Input() content;
   @Output() clicked: EventEmitter<any> = new EventEmitter();
+  @Input() isDisabled = false;
   @ViewChild('elementref') elementref;
   @ViewChild('contentref') contentref;
+
+  emitClickEvent($event: Event): void {
+    this.clicked.emit($event);
```
Comment (GustavoCostaW) [inline components/ngc-float-item-button.component.ts]: for what reason you create the `emitClickEvent()` method in the component but 

```
(click)="clicked.emit($event)"> 
```

Is more simplified and also works, right?

File: components/ngc-float-item-button.component.ts [hunk 2]
```diff
@@ -63,12 +69,12 @@ import {
 
   `],
   template: `
-    <div #elementref class="item" (click)="clicked.emit($event)">
+    <div #elementref class="item {{ isDisabled ? 'disabled' : ''}}" (click)="emitClickEvent($event)">
```
Comment (GustavoCostaW) [inline components/ngc-float-item-button.component.ts:72]: Okay, the interpolation add the class if isDisabled exist, but the component with isDisabled property if clicked still dispatch the events in the clicked observable.

this is stranger, what you think about?

**Review by GustavoCostaW — CHANGES_REQUESTED** [review]

**Comment by GustavoCostaW** [comment]
@kreatemore hey, thank you for the PR, I added some comments there.

File: components/ngc-float-item-button.component.ts (hunk 2 above)
Comment (duck-nukem) [reply components/ngc-float-item-button.component.ts:72]: The class `disabled` has `pointer-events` set to `none`, so it really shouldn't pick it up, I've now added an early exit to the emit function, to be sure that nothing will get through.

File: components/ngc-float-item-button.component.ts (hunk 1 above)
Comment (duck-nukem) [reply components/ngc-float-item-button.component.ts]: I prefer to have a method with a name that implies what the function is going to do - and have the actual implementation inside the component's class, not the HTML.

File: components/ngc-float-item-button.component.ts (hunk 2 above)
Comment (GustavoCostaW) [reply components/ngc-float-item-button.component.ts:72]: Great!

**Review by GustavoCostaW — APPROVED** [review]

**Comment by GustavoCostaW** [comment]
merged, thx @kreatemore

**Pull request merged**